tests/unittests/torch_dataset_tests.py

```
@@ -167,6 +237,188 @@ def test_skip_failures(self):
+
+            gi = RequireLabel(
+                field_mapping={"label": "ground_truth.detections.label"}
+            )
+
+            # Without skip_failures, the failing row raises
+            td = dataset.to_torch(
+                gi,
+                index_field="ground_truth.detections.id",
+                skip_failures=False,
+            )
+            with self.assertRaises(Exception):
```

Ruff (0.15.14)

[warning] Use `pytest.raises` instead of unittest-style `assertRaises`

Replace `assertRaises` with `pytest.raises`

(PT027)